asciidoc-confluence-publisher-doc/etc/docs/00-index/05-tables.adoc

```diff
@@ -101,3 +101,61 @@ Currently, only simple formattings, but no nested tables or lists are supported.
 | *B1*
 | *B2*
 |===
+
+== Table with Row Span
```

Comment:
First putting the col span, than the row span example seems more natural to me (as I expect col spans to be use more frequently).